library/newrelic-usage/dashboards/nr_usage_sub_account.json

```diff
@@ -0,0 +1,170 @@
+{
+  "name": "New Relic Usage (SubAcct)",
+  "description": "",
+  "pages": [
+    {
+      "name": "New Relic Usage (SubAcct)",
+      "description": "",
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 3,
+            "width": 6
+          },
+          "title": "",
+          "rawConfiguration": {
+            "text": "# Understanding your usage data in New Relic One\n\nThese queries include common scenarios for counting usage and estimating cost across New Relic One. \n\nTo set usage alerts to manage spend, visit our [docs page](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-users/usage-queries-alerts).\n\nFor month-to-date usage averages and cost, visit the [Usage UI](https://one.newrelic.com/launcher/usage-ui.launcher) in New Relic One.\n\nTo understand data ingest by event, type, or integration source, explore the [Data Management Hub](https://one.newrelic.com/launcher/data-management-nerdlets.data-management-launcher).\n\n"
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 7,
+            "row": 1,
+            "height": 3,
+            "width": 3
+          },
+          "title": "How many GBs have I accrued this month?",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "from NrConsumption select sum(GigabytesIngested) as 'Total GBs Ingested this Month' where productLine = 'DataPlatform' SINCE this month"
+              }
+            ],
+            "thresholds": []
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 10,
+            "row": 1,
+            "height": 3,
+            "width": 3
+          },
+          "title": "How many GBs have I accrued since my contract start date?",
```

Comment:
Consider
```suggestion
          "title": "How many GBs have I accrued since this year? (update query to reflect contract start date, if desired)",
```